mmv1/third_party/terraform/utils/bootstrap_utils_test.go

```diff
@@ -418,12 +418,17 @@ func removeContainerServiceAgentRoleFromContainerEngineRobot(t *testing.T, proje
 	}
 }
 
-func BootstrapProject(t *testing.T, projectID, billingAccount string, services []string) *cloudresourcemanager.Project {
+// BootstrapProject will create or get a project named "<projectIDPrefix><testProjectFromEnv>"
+// that will persist across test runs. The reason for the naming is to isolate
+// bootstrapped projects by test environment.
+func BootstrapProject(t *testing.T, projectIDPrefix, billingAccount string, services []string) *cloudresourcemanager.Project {
 	config := BootstrapConfig(t)
 	if config == nil {
 		return nil
 	}
 
+	projectID := fmt.Sprintf("%s%s", projectIDPrefix, getTestProjectFromEnv())
```

Comment:
Since we're just concatenating strings here, how about we make it
```
projectID := projectIDPrefix + getTestProjectFromEnv()
```

Reply:
Makes sense, will change